Sources/PostgresNIO/New/Connection State Machine/ConnectionStateMachine.swift

```diff
@@ -226,15 +229,15 @@ struct ConnectionStateMachine {
         }
     }
     
-    mutating func sslSupportedReceived(unprocessedBytes: Int) -> ConnectionAction {
+    mutating func sslSupportedReceived(unprocessedBytes: Int, promise: EventLoopPromise<Void>?) -> ConnectionAction {
```

Comment:
This function is only used from channel-reads so currently it'll never have a non-nil promise.
I'm still properly accepting and passing the possible-promise around.